Dockerfile

```diff
@@ -4,12 +4,22 @@ ARG PYTHON=3.11.7
 ARG TORCH=2.1.2
 ARG TORCH_REQUIREMENT="torch==${TORCH}"
 ARG EXTRA_INDEX_URL
+ARG TORCH_WHEEL_SOURCE="okoko/torch-wheels:2.1.2"
+
 ARG CREATED
 ARG SOURCE_COMMIT
 ARG CONSTRAINTS=constraints.txt
 
+# Cannot use ARG directly in 'RUN --mount ..', so use this stage as an alias
```

Comment:
Could document instead

    # Using variable in RUN --mount=from gives error 'from' doesn't support variable expansion, define alias stage instead